pkg/management/endpoint.go

```diff
@@ -156,13 +158,15 @@ func MakeGetUserEndpoint(managementComponent ManagementComponent) endpoint.Endpo
 func MakeUpdateUserEndpoint(managementComponent ManagementComponent) endpoint.Endpoint {
 	return func(ctx context.Context, req interface{}) (interface{}, error) {
 		var m = req.(map[string]string)
-
-		userJson := []byte(m["body"])
+		var err error
 
 		var user api.UserRepresentation
-		err := json.Unmarshal(userJson, &user)
 
-		if err != nil {
+		if err = json.Unmarshal([]byte(m["body"]), &user); err != nil {
+			return nil, err
+		}
+
+		if err = user.Validate(); err != nil {
```

Comment:
Yes, this is a bit a temporary fix. I have another JIRA regarding some input validation that currently returns 500 instead of BadRequest. So I did it in a quick way and planned to fix globally once common is merged. (I think our error management could be improved)